src/components/Share.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+
+class Share extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      
+    };
+   
+  }
+
+  render() {
+    return (
+      <div class="fb-share-button" data-href="https://espn.com" data-layout="button" data-size="large">
```

Comment:
I'm not sure this feature is useful for us. I'm happy you challenges yourself technically but this would only be useful, in my opinion, if we were able to get deeplinks on the app side to work.As it stands, I think we don't have enough time to get that done.

Reply:
cool and totally fine with me.

Reply:
We'll keep this for now as a way for people ( and us) to share the portal with other people.